packages/FileFormatter/Formatter/XmlFileFormatter.php

```diff
@@ -54,7 +54,7 @@ public function format(File $file, EditorConfigConfiguration $editorConfigConfig
         $this->padChar = $editorConfigConfiguration->getIndentStyleCharacter();
         $this->indent = $editorConfigConfiguration->getIndentSize();
 
-        $newFileContent = $this->formatXml($file->getFileContent());
+        $newFileContent = $this->formatXml($file->getFileContent(), $editorConfigConfiguration);
 
         $newFileContent .= $editorConfigConfiguration->getFinalNewline();
```

Comment:
the config was already used for the final newline, therefore I figured it actually should be used for all lines, not just the last one

Reply:
@staabm Looks good and consistent.